hack/staging-import-restrictions.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/apimachinery",
+    "allowedImports": [
+      "k8s.io/kube-openapi"
+    ]
+  },
+  {
+    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/api",
+    "allowedImports": [
+      "k8s.io/apimachinery"
+    ]
+  },
+  {
+    "baseImportPath": "k8s.io/kubernetes/vendor/k8s.io/kube-gen",
```

Comment:
we need:
* allow gengo, kube-openapi in k8s.io/kubernetes/vendor/k8s.io/kube-gen
* allow apimachinery, client-go in k8s.io/kubernetes/vendor/k8s.io/kube-gen/test **in addition**